frontend/public/components/marketplace/kubernetes-marketplace.jsx

```diff
@@ -101,11 +114,23 @@ MarketplaceListPage.propTypes = {
 
 export const Marketplace = () => {
   const resources = [];
+  resources.push({
+    isList: true,
+    kind: referenceForModel(CatalogSourceConfigModel),
+    namespace: 'openshift-operators',
+    prop: 'catalogsourceconfigs',
+  });
+  resources.push({
+    isList: true,
+    kind: referenceForModel(OperatorGroupModel),
+    prop: 'operatorgroups',
+  });
   resources.push({
     isList: true,
     kind: referenceForModel(PackageManifestModel),
-    namespace: 'openshift-marketplace',
+    namespace: 'openshift-operators',
     prop: 'packagemanifests',
+    // selector: {matchExpressions: [{key: 'openshift-marketplace'}]},
```

Comment:
I think this comment was left accidentally

Reply:
good catch! For testing purposes I had to comment it out since the code that provides the label was not yet merged

Reply:
Got it; I should have recognized that label